<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="to_vk_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>VK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="to_tg_btn">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Telegram</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="to_inst_btn">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Instagram</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="to_twit_btn">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Twitter - X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="to_face_btn">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Facebook</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
